fbw-common/src/wasm/systems/systems/src/electrical/battery_charge_rectifier_unit.rs

```diff
@@ -72,7 +76,7 @@ impl BatteryChargeRectifierUnit {
         self.battery_pb_is_auto = battery_push_buttons.bat_is_auto(self.number);
         let ac_bus_powered = electricity.is_powered(self);
 
-        self.loss_of_ac_duration = if ac_bus_powered {
+        self.loss_of_ac_duration = if ac_bus_powered || self.ground_servicing && !ground_servicing {
```

Comment:
The naming here makes it a little hard to read. Maybe an expression like `let ground_servicing_turned_off = self.ground_servicing && !ground_servicing;` would improve it.

Reply:
Thanks for the suggestion.
`ground_servicing` means that ground servicing mode is enabled. Currently this means that the contactor connecting the AC GND SVC bus with TR2 is closed.

About the logic: Good point. I can't remember exactly why it's needed. I'll do some more tests to be sure it's needed.

Reply:
Since it's working I'll merge it as is. I'll have a look at the logic at a later point.